src/ranged.cpp

```diff
@@ -929,33 +956,51 @@ static int do_aim( player &p, const std::vector<Creature *> &t, int cur_target,
     return cur_target;
 }
 
+static double approx_hit_chance( double dispersion, double range, double missed_by ) {
+    // This is essentially the inverse of what Creature::projectile_attack() does.
+
+    double missed_by_tiles = missed_by * occupied_tile_fraction;
+
+    //          T = (D**2 * (1 - cos V)) ** 0.5   (from iso_tangent)
+    //      cos V = 1 - T**2 / (2*D**2)
+    // 1 - V**2/2 = 1 - T**2 / (2*D**2)           (small angle approximation)
+    //          V = T / D
+    double shot_dispersion = ( range < 0.1 ? M_PI : missed_by_tiles / range ) * 180 * 60 / M_PI;
+    double sigma = dispersion / dispersion_sigmas;
+
+    // erf(x / (sigma * sqrt(2))) is the probability that a measurement
+    // of a normally distributed variable with standard deviation sigma
+    // lies in the range [-x..x]
+    return std::erf( shot_dispersion / ( sigma * 1.4142 ) );
```

Comment:
I guess it is a libstdc++ problem. What's the platform exactly? It works OK here on `gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.3)` with `libstdc++ 4.8.4-2ubuntu1~14.04.3`.

I can drop the std:: if the un-namespaced function is available everywhere